src/mito2/src/compaction/twcs.rs

```diff
@@ -353,7 +348,6 @@ impl CompactionTask for TwcsCompactionTask {
                     "Compacted SST files, input: {:?}, output: {:?}, window: {:?}",
                     added, deleted, self.compaction_time_window
                 );
-                self.early_success();
```

Comment:
Why remove this line?

Reply:
The behavior is incorrect. We should not send Ok here as the compaction is not finished yet.